src/filemap.rs

```diff
@@ -85,21 +85,38 @@ pub fn write_file(text: &StringBuffer,
         }
     }
 
+    fn diff(text: &StringBuffer,
+            filename: &str,
+            config: &Config)
+            -> Result<(String, String), io::Error> {
+        let mut f = try!(File::open(filename));
+        let mut ori_text = String::new();
+        try!(f.read_to_string(&mut ori_text));
+        let mut v = Vec::new();
+        try!(write_system_newlines(&mut v, text, config));
+        let fmt_text = String::from_utf8(v).unwrap();
+        Ok((ori_text, fmt_text))
```

Comment:
To be more precise this line used to call make diff, now it just returns the values, I'm open to a better name for the function